src/components/about-us/AboutUs.jsx

```diff
@@ -0,0 +1,28 @@
+import './AboutUs.css';
+
+const AboutUs = () => {
+	return (
+		<div className="aboutUsContainer">
+			<img className="aboutUsUpperDivider" src="../assets/about-us/about-us-upper-divider.png" alt="" />
```

Comment:
ezeknél ne legyen relativ hivatkozás csak simán /assets/about-us/about-us-upper-divider.png
illetve, ha mobilra is lövünk nem lenne jobb a <picture> az <img> helyett?